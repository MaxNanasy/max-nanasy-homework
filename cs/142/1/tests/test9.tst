main () {        if (a[3] < 67 db4) {	print(a);	}	}