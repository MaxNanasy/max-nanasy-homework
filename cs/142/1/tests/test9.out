 MAIN  (  )  {  IF  (  ident  [  number  ]  <  number  ident  )  {  PRINT  (  ident  )  ;  }  }  <EOT> 